app/src/main/java/com/example/myapplication/RegisterFragment.kt

```diff
@@ -34,7 +37,7 @@ class RegisterFragment : Fragment() {
 
         _binding = FragmentRegisterBinding.inflate(inflater, container, false)
         val fido2ApiClient = Fido.getFido2ApiClient(this.requireContext().applicationContext)
-        val options = PasswordlessOptions("", "", "", "")
+        val options = PasswordlessOptions("nodejsdemobackend:public:a02aa4cca4ad464cb5acc50dcca33b37", "localhost", "https://demo-backend.passwordless.dev/", "https://v4.passwordless.dev")
```

Comment:
Can we move this into a configuration file, so it's easy to change?

The final parameter is optional, you might not have to specify 'https://v4.passwordless.dev'